src/main/java/seedu/address/model/person/Name.java

```diff
@@ -9,14 +9,16 @@
  */
 public class Name {
 
-    public static final String MESSAGE_CONSTRAINTS =
-            "Names should only contain alphanumeric characters and spaces, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Names can include alphabetic characters, spaces, parentheses, "
+            + "dashes, slashes, at symbol(@), and commas.\n"
+            + "It must contain at least 1 alphabet character and has a character limit of 50.\n"
+            + "It cannot be empty and must not contain any other special characters or digits.";
 
     /*
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    public static final String VALIDATION_REGEX = "^(?=.*[a-zA-Z])[a-zA-Z ()/,@-]{1,50}$";
```

Comment:
Should we allow '@' in names?

Reply:
I think it doesn't hurt to allow more inputs

Reply:
based on singapore's general naming conventions there are instances where @ exists in someones official name